weetweet.py

```diff
@@ -447,50 +489,58 @@ def connect():
 
     # These arguments are optional. But the current code only handles this
     # configuration. So it's defined here if the defaults change.
-    stream_options = dict( timeout=None, block=True, heartbeat_timeout=90 )
+    stream_options = dict(timeout=None, block=True, heartbeat_timeout=90)
 
     # Reconnect timer list [1sec, 5sec, 10sec, 1min, 2min]
-    re_timer = [1,2,10,60,120]
+    re_timer = [1, 2, 10, 60, 120]
     re_timer_idx = 0
 
+    kwargs = dict(tweet_mode='extended')
+    #kwargs['screen_name'] = script_options['screen_name']
+
     while True:
         try:
             if name == "twitter":
                 #home timeline stream
                 stream = TwitterStream(auth=OAuth(
-                        oauth_token, oauth_secret, CONSUMER_KEY, CONSUMER_SECRET),
-                        domain="userstream.twitter.com", **stream_options)
+                    oauth_token, oauth_secret, CONSUMER_KEY, CONSUMER_SECRET),
+                                       domain="userstream.twitter.com")
```

Comment:
This will be removed later so it doesn't really matter. But why did you remove the `stream_options` here?